src/Transport/Topology/MetaModel/DefaultNamespaceNameToConnectionStringMapper.cs

```diff
@@ -15,7 +15,7 @@ public DefaultNamespaceNameToConnectionStringMapper(ReadOnlySettings settings)
 
         public EntityAddress Map(EntityAddress value)
         {
-            if (!value.HasSuffix) return value;
+            if (!value.HasSuffix || value.HasConnectionString) return value;
```

Comment:
It isn't clear to me how `value` can have a connection string without having a suffix. Am I missing something?